From ada5836693eaee410dc939fd3ecf152b7a9c13ef Mon Sep 17 00:00:00 2001
From: Abraham Elmahrek <abraham@elmahrek.com>
Date: Wed, 5 Nov 2014 14:20:14 -0800
Subject: [PATCH 054/101] SQOOP-1663: OraOop test cases are not logging any output

---
 .../sqoop/manager/oracle/OraOopTestCase.java       |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/src/test/org/apache/sqoop/manager/oracle/OraOopTestCase.java b/src/test/org/apache/sqoop/manager/oracle/OraOopTestCase.java
index 27b53ee..6d0cf89 100644
--- a/src/test/org/apache/sqoop/manager/oracle/OraOopTestCase.java
+++ b/src/test/org/apache/sqoop/manager/oracle/OraOopTestCase.java
@@ -186,7 +186,6 @@ protected Configuration getSqoopConf() {
   protected int runImport(String tableName, Configuration sqoopConf,
       boolean sequenceFile) {
     Logger rootLogger = Logger.getRootLogger();
-    rootLogger.removeAllAppenders();
     StringWriter stringWriter = new StringWriter();
     Layout layout = new PatternLayout("%d{yy/MM/dd HH:mm:ss} %p %c{2}: %m%n");
     WriterAppender writerAppender = new WriterAppender(layout, stringWriter);
@@ -249,9 +248,6 @@ protected int runImport(String tableName, Configuration sqoopConf,
         rowsImported = Integer.parseInt(matcher.group(2));
       }
     }
-    if (retCode != 0 || rowsInTable != rowsImported) {
-      System.out.println(stringWriter.toString());
-    }
     Assert.assertEquals("Incorrect number of rows imported", rowsInTable,
         rowsImported);
     return retCode;
-- 
1.7.0.4

